docs/transparent-validation/muxpi/theory-of-operations.md

```diff
@@ -80,7 +82,7 @@ To check which IP address has been assigned, type in terminal: `ifconfig`.
 OPTIONAL STEP:
 To set static IP, create following file:
 
-`sudo vi /etc/NetworkManager/NetworkManager.conf`
+    sudo vi /etc/NetworkManager/NetworkManager.conf
```

Comment:
Code blocks should be added as follows:

```bash
<insert Your code here>
```
Please refer to another files in documentation.